<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>765</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>420</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateTimeEdit">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>420</y>
     <width>194</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="imglabel">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>90</y>
     <width>250</width>
     <height>250</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>360</y>
     <width>54</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>命卦</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
